src/id-verification/Camera.jsx

```
@@ -26,6 +31,87 @@ class Camera extends React.Component {
     this.cameraPhoto.stopCamera();
   }
 
+  startDetection() {
+    this.setState({ shouldDetect: !this.state.shouldDetect });
+    while (!this.state.hasLoaded) {
+      // wait until video has loaded
+    }
+    this.detectFromVideoFrame(this.props.model, this.videoRef.current);
+  }
+
+  detectFromVideoFrame = (model, video) => {
+    model.estimateFaces(video).then((predictions) => {
+      if (this.state.shouldDetect) {
+        this.showDetections(predictions);
+
+        requestAnimationFrame(() => {
+          this.detectFromVideoFrame(model, video);
+        });
+      }
+    }, (ignore) => {
+      // will catch error when next panel is reached, so we want to ignore
```

I also don't really like having an empty catch, but this throws an error once the "next" button is clicked on the panel. I think that's happening because this function is still running for a short time after the "next" button is clicked.